templates/master/01-master-kubelet/_base/files/kubelet.yaml

```diff
@@ -25,6 +25,7 @@ contents:
     serializeImagePulls: false
     staticPodPath: /etc/kubernetes/manifests
     systemCgroups: /system.slice
+    nodeStatusUpdateFrequency: 10s
```

Comment:
@cdoern , When this base kubelet template is already getting updated, I don't think we really need to make a change to this [case](https://github.com/openshift/machine-config-operator/pull/3784/files#diff-4a2d0a4442e10d889a927322ad7311c67b22b259622067b52cbda7d2ece3ebefR158) . WDYT?

Reply:
I think it might be best to leave the case for empty just in case a user somehow messes with their config?

Reply:
/hold
In my opinion, if you are already making a change to the base templates, you need not make any change to the `updateOriginalKubeConfigwithNodeConfig` function.
One of the reasons being:
Let's say in future, based on a new requirement (may be), we decide to set the `DefaultNodeStatusUpdateFrequency` to be `20s`, the `emptyInput` should set nothing and the underlying Kubelet config sets the `nodeStatusUpdateFrequency` to `10s`(as set in base template) and not `20s`
Basically, default `nodeStatusUpdateFrequency` need not depend on the `workerLatencyProfile` field's default value.

Any thoughts @harche @yuqi-zhang ?